features/cache_control_modification.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Proxy Modification of Cache-Control Header to include max-age when Babbage sets Cache-Control to "public" or "private"
+
+  Scenario Outline: Proxy adjusts Cache-Control header based on page release time context
```

Comment:
This isn't accurate. It adjusts the Cache-Control header when the Cache-Control header set by Babbage has certain values.

Reply:
Updated 👍